extension/lib/commands.coffee

```diff
@@ -509,6 +510,17 @@ helper_follow = ({name, callback}, {vim, count, callbackOverride = null}) ->
 
   vim._run(name, {pass: 'auto'}, injectHints)
 
+
+helper_add_user_context_id = (gBrowser, objectToAugment) ->
+  contextAttribute = gBrowser.selectedTab.attributes.getNamedItem(USER_CONTEXT_ID_ATTRIBUTE_NAME)
```

Comment:
`npm run gulp lint` complains:
>lib/commands.coffee
>  ✖  line 515  Line exceeds maximum allowed length  Length is 97, max is 80

```suggestion
  contextAttribute = gBrowser.selectedTab.attributes.getNamedItem(
    USER_CONTEXT_ID_ATTRIBUTE_NAME
  )
```

Reply:
oops will fix

Reply:
got this on my latest version - hope it's all good now.
```
↕️  3 VimFx on  feature/follow-tab-support-containers 
♖ ❯ npm run gulp lint

> VimFx@0.24.6 gulp ~/bench/cfg/VimFx
> gulp "lint"

[23:01:13] Requiring external module coffeescript/register
[23:01:13] Using gulpfile ~/bench/cfg/VimFx/gulpfile.coffee
[23:01:13] Starting 'lint'...
[23:01:13] Starting 'lint-workaround'...
[23:01:13] Finished 'lint-workaround' after 11 ms
[23:01:13] Starting 'lint-only'...
[23:01:14] Finished 'lint-only' after 1.38 s
[23:01:14] Finished 'lint' after 1.4 s
```